libraries/abstractions/wifi/test/iot_test_wifi.c

```diff
@@ -188,74 +208,44 @@ typedef struct
  */
 
 /* Custom WIFI Test asserts. */
-#define TEST_WIFI_ASSERT_EQ_REQUIRED_API( expected, actual, result )   \
-    if( ( result ) == eWiFiNotSupported )                              \
+#define TEST_WIFI_ASSERT_REQUIRED_API( condition, result )             \
+    if( result == eWiFiNotSupported )                                  \
     {                                                                  \
         TEST_FAIL_MESSAGE( "Required Wi-Fi API is not implemented." ); \
     }                                                                  \
     else                                                               \
     {                                                                  \
-        TEST_ASSERT_EQUAL_INT( expected, actual );                     \
-    }
-
-#define TEST_WIFI_ASSERT_NOT_EQ_REQUIRED_API( expected, actual, result ) \
```

Comment:
Looks like these macros are removed. Are they no longer used ?

Reply:
In the cherry-pick from share, usages of this macro had been replaced:
`TEST_WIFI_ASSERT_NOT_EQ_REQUIRED_API( a, b, c )` --> `TEST_WIFI_ASSERT_REQUIRED_API( a != b, c )`